security/tutorials/calico-policy.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Calico policy tutorial
+description: Learn how to create more advanced Calico network policies (namespace, allow and deny all ingress and egress).
+canonical_url: "/security/tutorials/calico-policy-advanced"
+---
+
+Calico Network Policies _extend_ the functionalities of Kubernetes Network Policies. To demonstrate this, this page will walk through the [Kubernetes Advanced NetworkPolicy](kubernetes-policy-advanced) tutorial but with everything implemented using Calico policies. This serves as a good example in highlighting the syntactical differences between the two type of policies while demonstrating Calico Network Policy's flexibility.
```

Comment:
Calico network policies extend the functionalities of Kubernetes network policies. To demonstrate this, this tutorial takes the [Kubernetes Advanced NetworkPolicy tutorial](kubernetes-policy-advanced), and implements it using Calico network policies. It not only highlights the syntactical differences between the two policy types, but also demonstrates the flexibility of Calico network policy.

Reply:
Changed! Thank you for the review! :D

Reply:
Fixed, thank you for the review! :D